Docs/docs.html

```diff
@@ -55,8 +55,90 @@ <h2>Usage</h2>
 
         <section id="Alerts">
             <h2>Alerts</h2>
-            <p></p>
-            <code></code>
+            <p>Alerts are used to communicate the user about any event</p>
+            <div class="alert-display-container">
+            <div class="alert alert-success">
+                <div class="alert-icon icon-success">
+                    <i class="fas fa-check fa-2x"></i>
+                </div>
+                <div class="alert-text">
+                    This is a Success alert!
+                </div>
+            </div>
+            <br>
+            <div class="alert alert-faliure">
+                <div class="alert-icon icon-faliure">
+                    <i class="fas fa-times fa-2x"></i>
+                </div>
+                <div class="alert-text">
+                    This is a Faliure alert!
+                </div>
+            </div>
+            <br>
+            <div class="alert alert-warning">
+                <div class="alert-icon icon-warning">
+                    <i class="fas fa-exclamation fa-2x"></i>
+                </div>
+                <div class="alert-text">
+                    This is a Warning alert!
+                </div>
+            </div>
+            <br>
+            <div class="alert alert-general">
+                <div class="alert-icon icon-general">
+                    <i class="fas fa-bell fa-2x"></i>
+                </div>
+                <div class="alert-text">
+                    This is a General alert!
+                </div>
+            </div>
+            <br>
+        </div>
+            <div class="alert-code-containers">
+            <pre><code>
```

Comment:
Any specific reason why you want to use pre and code combination instead of carbon snippets ? 
Benefit of carbon over this is that auto copy button, a better UX.

Reply:
Yes, Carbon definitely has its cool UI but,
I read that too many iframes slow the page load time.
since I'm keeping all my components and code on 1 page.
So I decided to go with highlightJS (as of now) and try to make it look better later
Also, I'm planning to add copy buttons.

For reference: 
https://u-and-i.netlify.app/docs.html 
takes more than 10s to display the codes on the page
and around 50s to completely load the page
you can check this in dev tools > network

Reply:
Sounds right! Have the same structure too, let see how it rules out.